specification/web/resource-manager/Microsoft.Web/stable/2024-04-01/CommonDefinitions.json

```diff
@@ -3007,6 +3007,10 @@
               "description": "client certificate authentication comma-separated exclusion paths",
               "type": "string"
             },
+            "endToEndEncryptionEnabled": {
```

Comment:
>endToEndEncryptionEnabled [](http://example.com/codeflow?start=13&length=25)

In place of adding a boolean flag you should use an enum like EncryptionType with values like [EndToEnd, None], in this way if in the future versions you are enabling another type of encryption it will just add a new value to the enum in place of other properties

Reply:
The feature is already implemented in our backend using a boolean. We are happy with it this way.